bower.json

```diff
@@ -20,16 +20,15 @@
   "ignore": [
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#0.8-preview",
-    "layout": "Polymer/layout",
-    "paper-styles": "PolymerLabs/paper-styles"
+    "polymer": "Polymer/polymer#v0.8.0-rc.4",
+    "paper-styles": "PolymerElements/paper-styles#^0.8.0"
   },
   "devDependencies": {
-    "webcomponentsjs": "Polymer/webcomponentsjs",
-    "core-doc-viewer": "Polymer/core-doc-viewer#0.8-preview",
-    "core-icon": "Polymer/core-icon#0.8-preview",
-    "core-icons": "Polymer/core-icons#0.8-preview",
-    "test-fixture": "Polymer/test-fixture#master",
-    "web-component-tester": "Polymer/web-component-tester#^2.2.0"
+    "iron-icon": "PolymerElements/iron-icon#^0.8.0",
+    "iron-icons": "PolymerElements/iron-icons#^0.8.0",
+    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.8.0",
+    "test-fixture": "PolymerElements/test-fixture#^0.8.0",
+    "web-component-tester": "Polymer/web-component-tester#^2.2.0",
+    "webcomponentsjs": "Polymer/webcomponentsjs#^0.6.0"
```

Comment:
Oh, seems like the repo was moved and it redirects. I'll change to `webcomponents/webcomponentsjs`. I'm old.